com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStreamDescriptor.cs

```diff
@@ -27,19 +27,23 @@ public AlembicStreamSettings Settings
         }
 
         [SerializeField] float abcStartTime = float.MinValue;
-        public float mediaStartTime
+        public float MediaStartTime
         {
             get => abcStartTime;
-            internal set => abcStartTime = value;
+            set => abcStartTime = value;
         }
 
         [SerializeField]
         float abcEndTime = float.MaxValue;
-        public float mediaEndTime
+        public float MediaEndTime
```

Comment:
Why is this now uppercase? That's a breaking API change. Same with `mediaStartTime` and `mediaDuration`.

Reply:
This is not a public method: internal class with public members are the norm :)
I made them uppercase for the sake of consistency (also made my serialized variables easier to name and use in expressions.)